cmake/Modules/CommonConfig.cmake

```diff
@@ -101,6 +101,10 @@ if (MSVC)
     endif()
 endif()
 
+if (DBUS AND (CMAKE_SYSTEM_NAME STREQUAL "Linux"))
```

Comment:
There is no need:
https://github.com/qbittorrent/qBittorrent/blob/4a66d705b80a73c34ab06867419190d624d53546/CMakeLists.txt#L33-L34

It can be fixed later if problem arises.